src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -0,0 +1,160 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { KeyCode } from 'vs/base/common/keyCodes';
+import { Disposable } from 'vs/base/common/lifecycle';
+import { URI } from 'vs/base/common/uri';
+import { IEditorConstructionOptions } from 'vs/editor/browser/config/editorConfiguration';
+import { EditorExtensionsRegistry } from 'vs/editor/browser/editorExtensions';
+import { CodeEditorWidget, ICodeEditorWidgetOptions } from 'vs/editor/browser/widget/codeEditorWidget';
+import { ITextModel } from 'vs/editor/common/model';
+import { IModelService } from 'vs/editor/common/services/model';
+import { LinkDetector } from 'vs/editor/contrib/links/browser/links';
+import { localize } from 'vs/nls';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { IContextViewDelegate, IContextViewService } from 'vs/platform/contextview/browser/contextView';
+import { IInstantiationService, createDecorator } from 'vs/platform/instantiation/common/instantiation';
+import { SelectionClipboardContributionID } from 'vs/workbench/contrib/codeEditor/browser/selectionClipboard';
+import { getSimpleEditorOptions } from 'vs/workbench/contrib/codeEditor/browser/simpleEditorOptions';
+import { IDisposable } from 'xterm';
+
+interface IAccessibleContentProvider { id: string; provideContent(): string; onClose(): void; options?: IAccessibleViewOptions }
+export const IAccessibleViewService = createDecorator<IAccessibleViewService>('accessibleViewService');
+
+export interface IAccessibleViewService {
+	readonly _serviceBrand: undefined;
+	show(providerId: string): void;
+	registerProvider(provider: IAccessibleContentProvider): void;
+}
+
+export interface IAccessibleViewOptions {
+	isHelpMenu: boolean;
+}
+
+export class AccessibleViewService extends Disposable implements IAccessibleViewService {
+	_serviceBrand: undefined;
```

Comment:
This should result in no emitted code:

```suggestion
	declare readonly _serviceBrand: undefined;
```